docs/_static/nidigital_usage.inc

```diff
@@ -3,13 +3,25 @@ Usage
 
 The following is a basic example of using the **nidigital** module to open a session to a ....
 
+.. code-block:: python
+
+    import nidigital
+    with nidigital.Session(resource_name='PXI1Slot2', channels='0') as session:
+        pass
+
+
+Some repeated capabilities can be nested. This is useful for some methods that can be used with the `pins`
+repeated capability. They can be nested with the `sites` repeated capability.
+
 .. code-block:: python
 
     import nidigital
     # Configure the session.
 
     with nidigital.Session(resource_name='PXI1Slot2', channels='0') as session:
-        pass
+        session.sites[0, 1].pins['PinA', 'PinB'].ppmu_source()
+
+This will expand the repeated capabilities string given to the driver as `'site0/2,site0/3,site1/2,site1/3'`
```

Comment:
Instead of talking in terms of what the driver runtime will be given, we should talk in terms of what the customer wants to achieve. Also, the expansion is incorrect!

It should say something like:

This will apply the method/property to `site0/PinA`, `site0/PinB`, `site1/PinA`, `site1/PinB`

Assuming or requiring the Python user to know about the C API is wrong.